certification/internal/bundle/bundle.go

```diff
@@ -166,10 +166,10 @@ func ExtractAnnotationsBytes(ctx context.Context, annotationBytes []byte) (map[s
 	return bundleMeta.Annotations, nil
 }
 
-func GetCsvFilePathFromBundle(mountedDir string) (string, error) {
+func GetCsvFilePathFromBundle(imageDir string) (string, error) {
 	log.Trace("reading clusterserviceversion file from the bundle")
-	log.Debug("mounted directory is ", mountedDir)
```

Comment:
Wow! Is this a remnant from the unshare days?

Reply:
Sure is.